src/components/pages/batch/list/index.tsx

```diff
@@ -1,51 +1,53 @@
-import { Box, Button, ButtonGroup, useDisclosure } from "@chakra-ui/react";
-import Accesser from "@components/@core/accesser";
-import CCMultiSelect from "@components/@core/accesser/cc-multi-select";
-import { CoreGrid, PageHeading } from "@components/@core/layout";
+import {
+  Box,
+  Button,
+  ButtonGroup,
+  Skeleton,
+  Table as ChakraTable,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+  useDisclosure,
+} from "@chakra-ui/react";
+import { PageHeading } from "@components/@core/layout";
 import Table from "@components/@core/table";
 import useGlobalState from "@hooks/use-global-state";
 import AddIcon from "@icons/add";
 import { Batch } from "@interfaces/traceability";
-import { BATCH_TYPE, ROLES } from "@static/constants";
-import { BATCH_CREATE, LOT_CREATE } from "@static/events";
+import { ROLES } from "@static/constants";
+import { LOT_CREATE } from "@static/events";
 import { hasAccess } from "@utils/auth";
 import React, { useEffect, useState } from "react";
 import { emit } from "react-gbus";
 import InfiniteScroll from "react-infinite-scroller";
 
-import { batchColumns } from "./data";
+import { createBatchColumns } from "./data";
 import BatchCreateModal from "./modals/batch-create-modal";
-import BatchUpdateModal from "./modals/batch-update-modal";
+import BatchUpdateModal from "./modals/batch-update-modal-new";
 import LotCreateModal from "./modals/lot-create-modal";
 import MultipleTypeWarning from "./multiple-warning";
 import { useBatchStore } from "./use-batch-store";
 
 function BatchListPageComponent() {
-  const [co, setCo] = useState({} as any);
-  const [ccs, setCCs] = useState([] as any);
+  const [ccs] = useState([] as any);
   const [ccCodes, setCCCodes] = useState<any>([]);
   const { state, ...actions } = useBatchStore();
   const { user } = useGlobalState();
   const [showTypeError, setShowTypeError] = useState(false);
   const [selectedBatches, setSelectedBatches] = useState<Required<Batch>[]>([]);
   const { isOpen: clearRows, onToggle } = useDisclosure();
-  const [hideAccessor, setHideAccessor] = useState<boolean>();
+  const [triggerRender, setTriggerRender] = useState(false);
 
   useEffect(() => {
-    ccCodes.length && actions.listBatch({ ccCodes, reset: true });
-  }, [ccCodes]);
+    actions.listBatch({ ccCodes: [71, 70, 78, 77, 73, 76, 72, 74, 69, 75], reset: true });
```

Comment:
is it necessary to hard code these values? is it possible to fetch  it from somewhere?